include/libtorrent/aux_/torrent_peer_allocator.hpp

```diff
@@ -84,13 +84,13 @@ namespace libtorrent {
 		// to have tens of thousands of peers, and a pool
 		// saves significant overhead
 
-		boost::pool<> m_ipv4_peer_pool{sizeof(libtorrent::ipv4_peer), 500};
-		boost::pool<> m_ipv6_peer_pool{sizeof(libtorrent::ipv6_peer), 500};
+		boost::pool<> m_ipv4_peer_pool{sizeof(libtorrent::aux::ipv4_peer), 500};
+		boost::pool<> m_ipv6_peer_pool{sizeof(libtorrent::aux::ipv6_peer), 500};
 #if TORRENT_USE_I2P
-		boost::pool<> m_i2p_peer_pool{sizeof(libtorrent::i2p_peer), 500};
+		boost::pool<> m_i2p_peer_pool{sizeof(libtorrent::aux::i2p_peer), 500};
 #endif
 #if TORRENT_USE_RTC
-		boost::pool<> m_rtc_peer_pool{sizeof(libtorrent::rtc_peer), 500};
+		boost::pool<> m_rtc_peer_pool{sizeof(libtorrent::aux::rtc_peer), 500};
```

Comment:
couldn't you just remove the `libtorrent::` qualifier instead? since they are in the same namespace now